node/node_test.go

```diff
@@ -103,7 +131,7 @@ func TestNodeDelayedStart(t *testing.T) {
 	now := tmtime.Now()
 
 	// create & start node
-	n, err := DefaultNewNode(config, ipfs.Mock(), log.TestingLogger())
```

Comment:
If that is the only purpose, we can remove the ipfsApi from the DefaultNewNode and make the defaul embed again.

Reply:
I'll update the PR later and add you as a reviewer. Maybe it becomes clearer then.